cfn-templates/msindexer.yaml

```diff
@@ -65,33 +65,44 @@ Resources:
                   - 'dynamodb:*'
                 Resource: !GetAtt YTMediaDDBQueueTable.Arn
   
-  PytubeLambdalayer:
-    Type: AWS::Lambda::LayerVersion
+  YTDLPLambdalayer:
+    Type: "AWS::Lambda::LayerVersion"
     Properties:
-      Content:
-        S3Bucket: '<ARTIFACT_BUCKET_TOKEN>'
-        S3Key: !Join
-          - ''
-          - - '<ARTIFACT_PREFIX_TOKEN>'
-            - '<PYTUBELLAYER_ZIPFILE>'
-      Description: Layer with pytube package required to download YT Videos
-      LayerName: pytube-llayer
+      Content: ../layers/yt_dlp
+      CompatibleRuntimes:
+        - python3.11      
+      Description: Layer with yt_dlp package required to download YT Videos
       LicenseInfo: MIT
+
+#  QBusinessModelLayer:
+#    Type: "AWS::Lambda::LayerVersion"
+#    Properties:
+#      Content: ../layers/qbusiness_boto3_model
+#      CompatibleRuntimes:
+#        - python3.11
+#      Description: Layer with AmazonQ boto3 model
+#      LicenseInfo: MIT
       
+  FFMPEGLambdalayer:
+    Type: "AWS::Lambda::LayerVersion"
+    Properties:
+      Content: ../layers/ffmpeg
+      CompatibleRuntimes:
+        - python3.11      
+      Description: Layer with ffmpeg package required to support yt_dlp functions on Lambda
+      LicenseInfo: MIT
+
   YouTubeVideoIndexer:
     Type: 'AWS::Lambda::Function'
     Properties:
       Layers: 
-          - !Ref PytubeLambdalayer
-      Code:
-        S3Bucket: '<ARTIFACT_BUCKET_TOKEN>'
-        S3Key: !Join
-          - ''
-          - - '<ARTIFACT_PREFIX_TOKEN>'
-            - '<YTINDEXER_ZIPFILE>'
+          - !Ref YTDLPLambdalayer
+          - !Ref FFMPEGLambdalayer
+#          - !Ref QBusinessModelLayer
```

Comment:
Not using QBusiness in this release

Reply:
Agreed. Removing commented line to avoid confusion